### ISSUE TITLE
This server is broken.

### ISSUE THREAD

**Issue by xiahualiu** [avivace/ror2-server#8]
```
Could not load config /Config/server_pregame.cfg: Could not find file "Z:\home\steam\ror2-dedicated\Risk of Rain 2_Data\Config\server_pregame.cfg"

(Filename: C:\buildslave\unity\build\Runtime/Export/Debug.bindings.h Line: 45)
```
The error is so devastating that I have to kill the process manually to exit.

**Comment by KyberJeffHason** [comment]
same problem

**Comment by dubvulture** [comment]
That's just a debug log, not an unrecoverable error (see other issues).
If you try to connect it should be working as intended.

As of the impossibility of using `ctrl+c` to kill the process, that's probably on SteamCMD.

**Comment by fnicolini** [comment]
First of all, @xiahualiu i edited the issue title because it really wasn't necessary the add that part (caps lock as well). Not only your title was pretty harsh for no particular reason, but the issue itself is useless and just demonstrates the fact that u didn't even bother checking the other issues for the answer to your "problem" (which really isn't one).

Here's the link to 2 different answer in another issue #3  that would solved your doubts:
[](https://github.com/avivace/ror2-server/issues/3#issuecomment-673642660)
[](https://github.com/avivace/ror2-server/issues/3#issuecomment-673715172)

In short, as @dubvulture said, it's simply an informatve log about the abscence of a file that is **not** necessary at all for the server to work.
For killing the container you should refer to the [docker ](https://docs.docker.com/engine/reference/commandline/kill/) documentation of the `docker kill` command and consider not running the container in foreground if you don't want your shell to be hanging.
